apps/condo/domains/common/components/PostMessageProvider/globalHandlers.tsx

```diff
@@ -242,6 +241,20 @@ export const useUpdateProgressBarHandler: () => RequestHandler<'CondoWebAppUpdat
     }, [userId])
 }
 
+export const useGetFragmentHandler: () => RequestHandler<'CondoWebAppGetFragment'> = () => {
+    return useCallback(() => {
+        if (typeof window === 'undefined') {
+            return { fragment: '' }
+        }
+
+        const fragment = window.location.hash.startsWith('#') 
```

Comment:
Is it possible not to start with `#`?

Reply:
Should we also `decodeURIComponent` here?

Reply:
<img width="390" height="45" alt="Image" src="https://github.com/user-attachments/assets/dea766bb-2e36-4583-bac8-efdf4670bfd5" />